problem.py

```diff
@@ -24,6 +24,8 @@ def _read_data(path, f_name):
     data = pd.read_csv(os.path.join(path, 'data', f_name))
     y_array = data[_target_column_name].values
     X_df = data.drop(_target_column_name, axis=1)
+    # parse the date column
+    X_df['DateOfDeparture'] = pd.to_datetime(X_df['DateOfDeparture'])
```

Comment:
doing this will break all previous submissions and it's a good for users to understand that this needs to be done both for X_df and external data. I would revert